src/steamship/cli/deploy.py

```diff
@@ -0,0 +1,218 @@
+import importlib
+import os
+import traceback
+import zipfile
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+import click
+from semver import VersionInfo
+
+from steamship import Package, PackageVersion, Steamship, SteamshipError
+from steamship.cli.manifest_init_wizard import validate_handle, validate_version_handle
+from steamship.cli.ship_spinner import ship_spinner
+from steamship.data.user import User
+from steamship.invocable.lambda_handler import get_class_from_module
+from steamship.invocable.manifest import Manifest
+
+DEFAULT_BUILD_IGNORE = [
+    "build",
+    ".git",
+    ".venv",
+    ".ipynb_checkpoints",
+    ".DS_Store",
+    "venv",
+    "tests",
+    "examples",
+    ".idea",
+    "__pycache__",
+]
+
+
+def update_config_template(manifest: Manifest):
+
+    path = Path("src/api.py")
+    if not path.exists():
+        path = Path("api.py")
+        if not path.exists():
+            raise SteamshipError("Could not find api.py either in root directory or in src.")
+
+    module = None
+    try:
+        module = importlib.machinery.SourceFileLoader("api", str(path)).load_module()
+    except Exception:
+        click.secho(
+            "An error occurred while loading your api.py to check configuration parameters. Full stack trace below.",
+            fg="red",
+        )
+        traceback.print_exc()
+        click.get_current_context().abort()
+
+    invocable_type = get_class_from_module(module)
+
+    config_parameters = invocable_type.config_cls().get_config_parameters()
+
+    if manifest.configTemplate != config_parameters:
+        if len(config_parameters) > 0:
+            click.secho("Config parameters changed; updating steamship.json.", fg="cyan")
+            for param_name, param in config_parameters.items():
+                click.echo(f"{param_name}:")
+                click.echo(f"\tType: {param.type}")
+                click.echo(f"\tDefault: {param.default}")
+                click.echo(f"\tDescription: {param.description}")
+        else:
+            click.secho("Config parameters removed; updating steamship.json.", fg="cyan")
+
+    manifest.configTemplate = config_parameters
+    manifest.save()
+
+
+def get_archive_path(manifest: Manifest) -> Path:
+    return Path(".") / "build" / "archives" / f"{manifest.handle}_v{manifest.version}.zip"
+
+
+def bundle_deployable(manifest: Manifest):
+    archive_path = get_archive_path(manifest)
+    archive_path.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
Do we want to delete the folder before we create it? Maybe I missed the line where you're doing it.

Reply:
The archive path is different for every version.  This mimics the typescript behavior.  It could clean up after itself better?